blockwork/build/transform.py

```diff
@@ -46,7 +46,7 @@ def init_interfaces(self):
     @functools.lru_cache()
     def output_interfaces(self):
         interfaces: dict[str, Any] = {}
-        for name, (direction, interface) in self._interfaces.items():
+        for name, (direction, _host, interface) in self._interfaces.items():
```

Comment:
Would it be worth encoding this middle argument as an enum called `Target` or `Pipe` - then one option could be `HOST` to direct towards the host, and `FLOW` could act as it currently does. That would give us the option over time to introduce more pipes/targets if we need them?